primary/Tenkuu Danzai Skelter Heaven (primary).ass

```diff
@@ -161,100 +159,89 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
-Comment: 0,0:01:43.35,0:01:47.95,Signs,,0,0,0,,# Engineered humans
-Dialogue: 0,0:01:43.35,0:01:47.95,Signs,,0,0,0,,{\pos(581.212,290.969)}Work was being done to engineer humans to follow all orders as soon as they are born.
-Comment: 0,0:01:47.85,0:01:49.36,Signs,,0,0,0,,# First generation
-Dialogue: 0,0:01:47.85,0:01:49.36,Signs,,0,0,0,,{\pos(322.545,320.302)}AA, after many years of development, had succeeded in producing the first generation of such beings.
-Comment: 0,0:01:48.45,0:01:49.25,Signs,,0,0,0,,# To prepare
-Dialogue: 0,0:01:48.45,0:01:49.25,Signs,,0,0,0,,{\pos(498.4,420)}To prepare for when that day comes...
-Comment: 0,0:01:53.03,0:02:07.57,Signs,,0,0,0,,# Identical programming
-Dialogue: 0,0:01:53.03,0:02:07.57,Signs,,0,0,0,,{\pos(69.212,197.635)}They were given the name "Weattprhopet." All of them were programmed identically, but their abilities had differentiated from each other over time.
-Comment: 0,0:02:07.87,0:02:29.13,Signs,,0,0,0,,# Training
-Dialogue: 0,0:02:07.87,0:02:29.13,Signs,,0,0,0,,{\pos(572,219.333)}Each Weattprhopet became one with a Skelter and trained with it constantly in preparation for that day.
-Comment: 0,0:03:12.20,0:03:14.57,Signs,,0,0,0,,# Heavenly Arbiter OP end
+Dialogue: 0,0:01:34.97,0:01:38.48,Signs,,0,0,0,,{\pos(320.571,164)}Heavenly Arbiter
+Dialogue: 0,0:01:40.05,0:01:40.48,Signs,,0,0,0,,{\pos(226.857,266.286)}Her fate which does not get used to by-nature\Nmachines was left to your hand...{jymmy edit: You hold the power to save her soul from her mechanical body's pre-programmed fate}
+Comment: 0,0:01:43.35,0:01:47.95,Signs,,0,0,0,,{\pos(342.545,73.636)}Specially Designated Public Corporation Altamira Agency{There's a typo in "Designated" (there's no 'a'), but otherwise this should be probably be left alone since SDPC is a real thing, otherwise known as a semigovernmental corporation. The initials SDPC appear later in the OP.}
+Dialogue: 0,0:01:43.35,0:01:47.95,Signs,,0,0,0,,{\pos(581.212,290.969)}The human body arms which had the program embedded\Nso that a command may be obeyed from from when born\N
+Dialogue: 0,0:01:47.85,0:01:49.36,Signs,,0,0,0,,{\pos(322.545,320.302)}AA passed many feature and years\Nand succeededin producing the first\Ngeneration at least
+Dialogue: 0,0:01:48.45,0:01:49.25,Signs,,0,0,0,,{\pos(498.4,420)}To the sake when coming
+Dialogue: 0,0:01:53.03,0:02:07.57,Signs,,0,0,0,,{\pos(69.212,197.635)}Even if they were called "Weattprhopet"{sic, from ウェットプロープ} and gave the complete same\Nprograms, become altogetherdifferent capability balance
+Dialogue: 0,0:02:07.87,0:02:29.13,Signs,,0,0,0,,{\pos(572,219.333)}She "Weattprhopet" got into "Skelter" united with each individuality, and has\Nrepeted{"repeated"} the improvement constantly in preparation for the day which should come.
```

Comment:
This still appears to be reverted to the pre-edit state, so make sure this doesn't make it as-is in when we merge it.